Problems publishing an exe file

Using this guide https://dotnetcoretutorials.com/2019/06/20/publishing-a-single-exe-file-in-net-core-3-0/

```ps
dotnet publish -c Release /p:PublishSingleFile=true -r win-x64
```

I obtained the error as shown below

```ps
Microsoft (R) Build Engine version 16.2.32702+c4012a063 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

C:\Sources\Neo\neo-devpack-dotnet\src\Neo.Compiler.MSIL\Neo.Compiler.MSIL.csproj : error NU1605: Detected package downgrade: System.IO.FileSystem.Primitives from 4.3.0 to 4.0.1. Reference the package directly from the project to select a different version.
C:\Sources\Neo\neo-devpack-dotnet\src\Neo.Compiler.MSIL\Neo.Compiler.MSIL.csproj : error NU1605:  Neo.Compiler.MSIL -> Mono.Cecil 0.10.3 -> System.IO.FileSystem 4.0.1 -> runtime.win.System.IO.FileSystem 4.3.0 -> System.IO.FileSystem.Primitives (>= 4.3.0)
C:\Sources\Neo\neo-devpack-dotnet\src\Neo.Compiler.MSIL\Neo.Compiler.MSIL.csproj : error NU1605:  Neo.Compiler.MSIL -> Mono.Cecil 0.10.3 -> System.IO.FileSystem.Primitives (>= 4.0.1)
  Restore failed in 594.52 ms for C:\Sources\Neo\neo-devpack-dotnet\src\Neo.Compiler.MSIL\Neo.Compiler.MSIL.csproj.
```

Also, we should provide releases for make this procedure easier for devs

--- a/src/Neo.Compiler.MSIL/Neo.Compiler.MSIL.csproj
+++ b/src/Neo.Compiler.MSIL/Neo.Compiler.MSIL.csproj
@@ -22,7 +22,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Mono.Cecil" Version="0.10.3" />
+    <PackageReference Include="Mono.Cecil" Version="0.11.0" />
     <PackageReference Include="Neo" Version="3.0.0-CI00172" />
   </ItemGroup>
 

